pmm-app/selenium.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+echo "Warning: you must have permission to run docker."
+docker pull selenoid/vnc_chrome:80.0
```

Comment:
@puneet0191 pls see if we can run the test on an earlier version of Chrome , say latest - 2 => 78